fuel-jackson-jvm/src/test/kotlin/fuel/jackson/FuelJacksonTest.kt

```diff
@@ -45,19 +51,14 @@ class FuelJacksonTest {
         }
 
         val response = Fuel.get(mockWebServer.url("user-agent").toString())
-        val jackson = response.toJackson<HttpBinUserAgentModel>(createCustomMapper())
+        val jackson = response.toJackson<HttpBinUserAgentModel>(createCustomMapper)
```

Comment:
[q] Just wondering whether we could provide a nice DSL for client to configure the mapper?

```kotlin
fun toJackson<T>(configure: ObjectMapper.() -> Unit) {
  val mapper = ObjectMapper()
  mapper.apply(configure)
}
```

Reply:
you could have to write a PR about it.